.gitignore

```diff
@@ -31,3 +31,6 @@
 
 # Ignore auto-generated translations.js
 /public/javascripts/translations.js
+
+# Secrets should be secret
+/config/secrets.yml
```

Comment:
Hmmm I think we need this [file](https://github.com/julianguyen/ifme/blob/master/config/secrets.yml) not to be secret though since we're getting `ENV["SECRET_KEY_BASE"]` from the .env files.